CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [12.1.0](https://github.com/Instabug/Instabug-Flutter/compare/v12.1.0...v11.14.0)
+
+### Added
+
+- Add support for Session Replay, which includes capturing session details, visual reproduction of sessions as well as support for user steps, network and Instabug logs. ([#395](https://github.com/Instabug/Instabug-Flutter/pull/395)).
+
+### Changed
+
+- **BREAKING** Remove deprecated APIs ([#385](https://github.com/Instabug/Instabug-Flutter/pull/385)). See migration guide for more details.
```

Comment:
Is the migration guide ready? If so, let's link to it.

Reply:
It's not ready yet, that's why we don't have a link to it in the changelog at the moment.

We chose to not wait on the migration guide to not delay the PR since all the other PRs were pending on it.